phpmyfaq/admin/assets/scss/_global.scss

```
@@ -1,117 +1,365 @@
+
+#faqpassword_toggle {
+  cursor: pointer;
+  color: var(--pmf-text-muted);
+
+  &:hover {
+    color: var(--pmf-accent);
+    background-color: var(--pmf-accent-subtle);
+  }
+
+  .bi {
+    color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)